lib/diagnostic.rb

```diff
@@ -142,4 +170,8 @@ def multiplier
       1
     end
   end
+
+  def get_possible_moves
+    'kick, move, punch'
+  end
```

Comment:
This should be `self.get_possible_moves` - as written, it's an instance method, not a class method.